### PR TITLE
Added tests for ActionsCore module

### PR DESCRIPTION
**Pull request by justinytchen**

### CODE REVIEW

File: tests/actionCore.tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,87 @@
+Import-Module -Name 'C:\Users\t-chenju\workspace\GitHub-Actions\PesterActionV1\lib\ActionsCore.psm1'
```
Comment (justinytchen) [inline tests/actionCore.tests.ps1:1]: need to change

File: tests/sample/actionCore.tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Describe "ActionOutputGroup" {
+    it "Constructs Enter command string correctly" {
+        Enter-ActionOutputGroup "name" | Should -Be "::group::name$([System.Environment]::NewLine)"
+    }
+
+    it "Constructs Exit command string correctly" {
+        Exit-ActionOutputGroup "name" | Should -Be "::endgroup::$([System.Environment]::NewLine)"
+    }
+}
```
Comment (TravisEz13) [inline tests/sample/actionCore.tests.ps1:87]:
```suggestion
}

```

Comment (TravisEz13) [reply tests/sample/actionCore.tests.ps1:87]: We should make sure we have newline at end of the file for posix 

See https://thoughtbot.com/blog/no-newline-at-end-of-file#history-lesson

File: tests/sample/actionCore.tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,87 @@
+Import-Module -Name 'PesterActionV1\lib\ActionsCore.psm1'
```
Comment (TravisEz13) [inline tests/sample/actionCore.tests.ps1:1]:
```suggestion
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT license.


Import-Module -Name 'PesterActionV1\lib\ActionsCore.psm1'
```

**Pull request merged**